tests/page-objects/forms/generic-form.po.js

```diff
@@ -1,6 +1,6 @@
 const helper = require('../../helper');
 const nameField = element(by.css('#report-form form [name="/data/name"]'));
-const submitButton = element(by.css('#report-form .submit'));
+const submitButton = element(by.css('.submit'));
```

Comment:
This selector feels fragile, like sometimes we might get multiple elements back. If you've checked it out and it's safe, then that's cool. If you aren't sure, are there options where we can add an `test-` id or class on part of the form to make it safer?

Reply:
I've added the `.enketo` class. That should restrict it to the submit button with forms. I don't recall any other submit buttons within forms.